### PR TITLE
PCP-1889: add disableLocalAccounts method and correct order of steps

### PR DESCRIPTION
**Pull request by benradstone**
## Describe the Change

This PR adds an additional step for disabling local accounts on the AKS cluster.

It also corrects the ordering of the **Deploy an Azure AKS Cluster** steps by moving one of the steps to an earlier stage.

## Changed Pages

💻 [Create and Manage Azure AKS Cluster](https://deploy-preview-5035--docs-spectrocloud.netlify.app/clusters/public-cloud/azure/aks/#deploy-an-azure-aks-cluster)

## Jira Tickets

🎫 [PCP-1889](https://spectrocloud.atlassian.net/browse/PCP-1889)

## Backports

Can this PR be backported?

- [x] Yes. _Remember to add the relevant backport labels to your PR._
- [ ] No. _Please leave a short comment below about why this PR cannot be backported._

[PCP-1889]: https://spectrocloud.atlassian.net/browse/PCP-1889?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docs-content/clusters/public-cloud/azure/aks.md [hunk 1]
```diff
@@ -108,7 +123,19 @@ explains how you can create an Azure AKS cluster managed by Palette.
...
+
+    ```yaml {7}
+    managedControlPlane:
+      aadProfile:
+        managed: true
+        adminGroupObjectIDs:
+          - <admin-group-object-id>
+          - <admin-group-object-id-2>
+        disableLocalAccounts: true
+    ```
```
Comment (benradstone) [inline docs/docs-content/clusters/public-cloud/azure/aks.md:136]: This is the new entry.

File: docs/docs-content/clusters/public-cloud/azure/aks.md [hunk 2]
```diff
@@ -74,7 +74,22 @@ explains how you can create an Azure AKS cluster managed by Palette.
 
 6.  Under **Managed Kubernetes**, select **Azure AKS** and select your Azure AKS cluster profile.
 
-7.  If you want to configure Pod and Service CIDR, populate the following configuration template and add the
+7.  Palette displays the cluster profile layers. Review the profile layers and customize parameters as desired in the
+    YAML files that display when you select a layer.
```
Comment (addetz) [inline docs/docs-content/clusters/public-cloud/azure/aks.md:78]: Do we need to say that they need to click on Values to see the YAML?

**Review by addetz — APPROVED** [review]
Nice work!

File: docs/docs-content/clusters/public-cloud/azure/aks.md (hunk 2 above)
Comment (benradstone) [inline docs/docs-content/clusters/public-cloud/azure/aks.md:78]:
```suggestion
7.  Palette displays the cluster profile layers. Review the profile layers and customize parameters as desired in the
    YAML editor that displays when you click on **Values** after selecting a layer.
```

Comment (benradstone) [reply docs/docs-content/clusters/public-cloud/azure/aks.md:78]: Good callout.

**Pull request merged**